cf/env.go

```diff
@@ -39,7 +38,7 @@ func SetCFOverrides(env env.Environment) error {
 
 		pgServiceName := cast.ToString(env.Get("storage.name"))
 		if pgServiceName == "" {
-			logrus.Warning("No PostgreSQL service name found")
+			log.D("cf/env").Warning("No PostgreSQL service name found")
```

Comment:
would it make sense that we do not specify the place where the logging occurs but it is transparent for the user of the facade? we could use a logic similar to https://github.com/onrik/logrus/blob/master/filename/filename.go#L52 or https://github.com/Peripli/service-broker-proxy/blob/master/pkg/logging/error_location_hook.go

Reply:
We can introduce the "hook", but I'd rather not keep state because then there would be the need of synchronization when working with these loggers.

Do you agree to introduce the management API and the caching of loggers in a separate PR? For this PR I can add the "hook"

Reply:
For the api i think its ok to put it in seperate pr - but if we decide to cache in order to avoid reflection repetition - i think it shouls be here (so that the logging change is self contained) - p.s. we havent decided anything yet 

About syncronization - it might not be necessary - no one will modify the base loggers after sm bootstrap - every filter that wants to add fields willwork with a copy of its (its packages?) base logger. So get from slice wouldnt require sync